cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/store/DatasetBasedTimeScheduleStore.java

```diff
@@ -95,7 +95,12 @@ public void storeTrigger(OperableTrigger newTrigger, boolean replaceExisting) th
   public void storeJobsAndTriggers(Map<JobDetail, Set<? extends Trigger>> triggersAndJobs,
                                    boolean replace) throws JobPersistenceException {
     super.storeJobsAndTriggers(triggersAndJobs, replace);
-    // TODO (sree): Add persisting job and triggers
+    for (Map.Entry<JobDetail, Set<? extends Trigger>> e: triggersAndJobs.entrySet()) {
+      storeJob(e.getKey(), replace);
+      for (Trigger trigger: e.getValue()) {
+        storeTrigger((OperableTrigger) trigger, replace);
+      }
+    }
```

Comment:
Is it better to have `storeJobTrigger(JobDetail newJob, OperableTrigger newTrigger, boolean replace)` method? This can be used then by the above code as well as by following `storeJobAndTrigger(JobDetail newJob, OperableTrigger newTrigger)`